### PR TITLE
Display Athletes on Home Page

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:
* Adds athletes show to dashboard/index page
* Adds show route 
* Adds feature test
* Adds basic steps.

#### What I have learned working on this feature: 
* To show a list of object on index page. 

#### Screenshots:
skjermbilde 2018-04-08 kl 12 21 12

### CODE REVIEW

**Review by larathordar — APPROVED** [review]
Looks good to me !

File: app/views/athletes/index.html.erb [hunk 1]
```diff
@@ -1 +1,9 @@
+<h1>Athletes</h1>
+
+<% Athlete.all.each do |athlete| %>
+  <athlete id="athlete">
```
Comment (holgertidemand) [inline app/views/athletes/index.html.erb]: Having this inside the loop will create a new "athlete" block for each the athletes. 
```
<ul id="athlete">
<% Athlete.all.each do |athlete| %>
 <li><%= link_to athlete.name, athlete_path(athlete)%></li>
<% end %>
</ul>
```

Might be a better way for managing the hole list.

File: app/views/athletes/index.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
+<h1>Athletes</h1>
+
+<% Athlete.all.each do |athlete| %>
```
Comment (holgertidemand) [inline app/views/athletes/index.html.erb]: Try to make this call in the controller instead and make use of instance variables in the view. 
```
def index
   @athletes = Athlete.all 
end 
```
and in the view. 
``` 
<% @athletes.each do |athlete| %>

```

That is better practice :)

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
   root controller: :athletes, action: :index
-  resources :athletes, only: [:index, :new, :create]
+  resources :athletes, only: [:index, :new, :create, :show]
```
Comment (holgertidemand) [inline config/routes.rb]: why did you add ``` :show ``` ? :) We are using the index action

**Review by holgertidemand — CHANGES_REQUESTED** [review]
Small things :) But looks good!

File: app/views/athletes/index.html.erb (hunk 2 above)
Comment (kmthorsnes) [reply app/views/athletes/index.html.erb]: Thank you. Fixed

File: config/routes.rb (hunk 3 above)
Comment (kmthorsnes) [reply config/routes.rb]: Fixed. I fix thoose two.

**Review by holgertidemand — COMMENTED** [review]
Just pull form upstream! Then we're good

**Review by holgertidemand — APPROVED** [review]
